templates/compose-datalake-dr.tmpl

```diff
@@ -20,6 +20,11 @@
             - cadence.enabled={{{get . "CADENCE_ENABLED"}}}
             - cadence.host={{{get . "CLUSTER_PROXY_CADENCE_HOST"}}}
             - cadence.port={{{get . "CLUSTER_PROXY_CADENCE_PORT"}}}
+        {{{- if get . "DATALAKE_DR_TARGET_PATH"}}}
+        volumes:
+        - {{{get . "DATALAKE_DR_TARGET_PATH"}}}/dependency-hard-links:/thunderhead-datalakedr-service/lib
+        - {{{get . "DATALAKE_DR_TARGET_PATH"}}}/thunderhead-datalakedr-service-0.1-SNAPSHOT.jar:/thunderhead-datalakedr-service/thunderhead-datalakedr-service-0.1-SNAPSHOT.jar
```

Comment:
wouldn't it make sense to use 2 different values? what if only one of them is wanted to be overridden? or the path on the host is different for some reason?

Reply:
I followed the exact logic currently used for the Datalake API, Environments2 API, and the Distrox API, so the decision of using 2 different values would need to be reflected in those files as well.

I agree that it's possible for the two paths to be different for some reason, but as of now all Thunderhead services have the same setups for their target files and their generated images.

Let me know what you think and thanks for looking at this!